tests/api/test_configurations.py

```diff
@@ -108,6 +108,12 @@ def test_post_configuration(
                 'mime_type': media_response['mime_type'],
             },
         },
+        headers={
+            "connect-auth": (
```

Comment:
For tests it is better to generate the test signature based on human-readable input. Otherwise hard to understand what is inside and what should be expected in the tests